azure/packages/azure-client/src/AzureClient.ts

```diff
@@ -210,11 +210,22 @@ export class AzureClient {
         };
 
         const codeLoader = { load };
+        const client: IClient = {
+            details: {
+                capabilities: { interactive: true },
+            },
+            permission: [],
+            scopes: [],
+            user: { id: "" },
+            mode: "write",
```

Comment:
Also is this change breaking in nature? Curious on your take.

Reply:
I don't think so because this object is copied from the default IClient object that only Container uses with mode modified

Reply:
This is a good read about our breaking-policy: https://github.com/microsoft/FluidFramework/wiki/Breaking-vs-Non-breaking-Changes. 

This is interesting area to consider. We are changing underlining behavior and that is usually considered a breaking change (as user needs to make efforts to adapt to the change), but in our case we may consider this as an optimization assuming that client code already should be able to handle reconnect flow.